<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppCfgWidget</class>
 <widget class="QWidget" name="AppCfgWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AppCfgWidget{
background-color:#F7F7F7;
}
QSplitter::handle:horizontal {
width: 1px;
background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(0, 0, 0, 30), stop:1 rgba(0, 0, 0, 255), stop:2 rgba(0, 0, 0, 30));
 }
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QScrollArea" name="scrollAreaNavi">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetNavi">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>250</width>
         <height>518</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frameSearch">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font-weight:bold;
font-size:14pt;
</string>
             </property>
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>251</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="lineEdit">
             <property name="placeholderText">
              <string>Search in settings</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameNaviBtns">
          <property name="styleSheet">
           <string notr="true">QPushButton{
border:none;
text-align:left;
padding:6px;
}
QPushButton:hover{
background-color:#EAEAEA;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnPersonal">
             <property name="text">
              <string>personal information </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnPrivacy">
             <property name="text">
              <string>Privacy Search and service</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAppearence">
             <property name="text">
              <string>Appearence</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnStartup">
             <property name="text">
              <string>Startup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTabPage">
             <property name="text">
              <string>New tab page</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDownload">
             <property name="text">
              <string>Download</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>385</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageAppearence">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>-60</y>
             <width>528</width>
             <height>633</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font-weight:bold;
font-size:13pt;
</string>
              </property>
              <property name="text">
               <string>Customize browser</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="styleSheet">
               <string notr="true">.QFrame{
border:1px solid gray;
background-color:white;
border-radius:8px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>defualt theme</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>328</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="2">
                <widget class="QComboBox" name="comboBox"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Zoom level</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>328</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="2">
                <widget class="QComboBox" name="comboBox_2"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">font-weight:bold;
font-size:13pt;
</string>
              </property>
              <property name="text">
               <string>Customize tool bars</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>500</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">.QFrame{
border:1px solid gray;
background-color:white;
border-radius:8px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QCheckBox" name="checkBox">
               <property name="geometry">
                <rect>
                 <x>442</x>
                 <y>15</y>
                 <width>16</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_6">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>108</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>show vertical tabs</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_7">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>39</y>
                 <width>120</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>show homepage button</string>
               </property>
              </widget>
              <widget class="QCheckBox" name="checkBox_2">
               <property name="geometry">
                <rect>
                 <x>442</x>
                 <y>44</y>
                 <width>16</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_8">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>68</y>
                 <width>78</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>set home page</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>98</y>
                 <width>91</width>
                 <height>20</height>
                </rect>
               </property>
              </widget>
              <widget class="QPushButton" name="pushButton">
               <property name="geometry">
                <rect>
                 <x>430</x>
                 <y>97</y>
                 <width>75</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>save</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_9">
               <property name="geometry">
                <rect>
                 <x>8</x>
                 <y>135</y>
                 <width>120</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>show bookmark bar</string>
               </property>
              </widget>
              <widget class="QComboBox" name="comboBox_3">
               <property name="geometry">
                <rect>
                 <x>440</x>
                 <y>130</y>
                 <width>69</width>
                 <height>20</height>
                </rect>
               </property>
              </widget>
              <widget class="QCheckBox" name="checkBox_3">
               <property name="geometry">
                <rect>
                 <x>440</x>
                 <y>170</y>
                 <width>16</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_10">
               <property name="geometry">
                <rect>
                 <x>8</x>
                 <y>165</y>
                 <width>120</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>show bookmark button</string>
               </property>
              </widget>
              <widget class="QCheckBox" name="checkBox_4">
               <property name="geometry">
                <rect>
                 <x>440</x>
                 <y>210</y>
                 <width>16</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_11">
               <property name="geometry">
                <rect>
                 <x>8</x>
                 <y>205</y>
                 <width>120</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>show downlaod button</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>6</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pagePrivacy"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
